bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Image.java

```diff
@@ -162,6 +169,26 @@ default String getTitle() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns the value for the image's {@code uuid} attribute, if one was set.
```

Comment:
I wouldn't mention the HTML attribute here, it's markup implementation specific.